service/raft/raft_group0.cc

```diff
@@ -824,7 +824,7 @@ future<group0_peer_exchange> raft_group0::peer_exchange(discovery::peer_list pee
             co_return group0_peer_exchange{group0_info{
                 .group0_id = std::get<raft::group_id>(_group0),
                 .id = _raft_gr.group0().id(),
-                .ip_addr = _raft_gr.address_map().get_inet_address(_raft_gr.group0().id()),
+                .ip_addr = _raft_gr.address_map().find(_raft_gr.group0().id()).value_or(gms::inet_address{}),
```

Comment:
Is it possible we're missing our own IP address here?

Reply:
the leader is not us necessarily. we might now it already and it would be some other node, which ip address is not yet known to us.

Reply:
But `_raft_gr.group0().id()` is not the leader's ID, it's our own ID.

Reply:
it's group 0 id, we're talking about server id.

Reply:
So do you suggest using get_broadcast_address instead?